### PR TITLE
chore(deps): extend tsconfig.json from tsconfigs

### PR DESCRIPTION
**Pull request by mightyiam**
#### Checklist

- [ ] I have read the documentation.
- [ ] I have included a pull request description of my changes.
- [ ] I have included some tests.
- [ ] If it's a new feature, I have included documentation updates.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,6 +87,7 @@
 		"tempy": "^0.3.0",
 		"tough-cookie": "^3.0.0",
 		"ts-node": "^8.5.2",
+		"tsconfigs": "^4.0.2",
```
Comment (mightyiam) [inline package.json]: https://www.npmjs.com/package/tsconfigs

File: tsconfig.json [hunk 2]
```diff
@@ -1,11 +1,9 @@
 {
-	"extends": "@sindresorhus/tsconfig",
+	"extends": "tsconfigs/nodejs-module",
```
Comment (mightyiam) [inline tsconfig.json]: https://github.com/mightyiam/tsconfigs/blob/v4.0.2/nodejs-module.json

File: tsconfig.json [hunk 3]
```diff
@@ -1,11 +1,9 @@
...
 	"compilerOptions": {
+		"skipLibCheck": true,
```
Comment (mightyiam) [inline tsconfig.json]: Required, else some of this project's `d.ts` files fail to compile. Which seems like an issue, itself. But a different issue.

https://github.com/sindresorhus/tsconfig/blob/v0.7.0/tsconfig.json#L24

**Comment by sindresorhus** [comment]
I appreciate the effort, but I'm very particular about my TypeScript config and prefer to have full control of it through my `tsconfig` package.

**Pull request closed without merging**